### PR TITLE
Add blogpost for sprint 134 SCM/CI integration

### PR DESCRIPTION
**Pull request by krauselukas**

### CODE REVIEW

File: _posts/development/2023-02-20-update-on-the-scm-integration.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+category: development
+---
+
+We took another step on the SCM/CI integration of the Open Build Service. This time
+we improved the rendering time of the workflow runs UI and enhanced the user
+documentation.
+
+### Reduced Rendering Time of Workflow Runs UI
+
+Previously with a lot of workflow run records being present, the UI took multiple
```
Comment (dmarcoux) [inline _posts/development/2023-02-20-update-on-the-scm-integration.md:13]: `records` sounds a bit too technical, I would go simply with `workflow runs`

File: _posts/development/2023-02-20-update-on-the-scm-integration.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Before: Loading time was > 6 seconds
+After: Loading time is < 1 second (~0.17 seconds)
+
+```
+
+### Use Case Documentation of Placeholder Variables and Configuration File URL
+
+We already created chapters in the user documentation around the placeholder variables
+and configuration file url. But we didn't provide a real life example of what is
+possible to do with it. We catched up on it and added a new section where we explain
```
Comment (dmarcoux) [inline _posts/development/2023-02-20-update-on-the-scm-integration.md:29]: [catched](https://conjugator.reverso.net/conjugation-english-verb-catch.html) doesn't exist:

```suggestion
possible to do with it. We caught up on it and added a new section where we explain
```

Comment (krauselukas) [reply _posts/development/2023-02-20-update-on-the-scm-integration.md]: @dmarcoux done

File: _posts/development/2023-02-20-update-on-the-scm-integration.md (hunk 1 above)
Comment (krauselukas) [reply _posts/development/2023-02-20-update-on-the-scm-integration.md]: done

**Review by danidoni — APPROVED** [review]

**Pull request merged**